ideascube/serveradmin/tests/data/a-video.mp4

```diff
@@ -0,0 +1 @@
+aaaaaaaaaaaaaaaaaaaaaaa
```

Comment:
The image/video/pdf come from tests for mediacenter import_medias command.
They are indeed duplicated but import_medias will be removed in the future (maybe now ?)